webrtc.html

```diff
@@ -2504,6 +2504,31 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpSender createRtpSender (DOMString kind)</dt>
+
+          <dd>
+            <p>Equivalent to
+            calling <a href="#dom-peerconnection-addtrack"><code>addTrack(null)</code></a>,
```

Comment:
I think that a type attribute would be a good idea given the possibility that we now make the only way of identifying the type (the track) optional.